client/assets/less/pages/place-detail.less

```diff
@@ -0,0 +1,202 @@
+.page-detail {
+  bottom: 0;
+  width: 100%;
+  top: 0;
+  background-color: #FFF;
+  .fa-spin-custom, .glyphicon-spin {
+    position: absolute;
+    top: 0;
+    left: 50%;
+    -webkit-animation: spin 1000ms infinite linear;
```

Comment:
You don't need use prefix -webkit. It's put automatically with autoprefix. Remove this in all places, where is it